.github/workflows/autoreview.yaml

```diff
@@ -30,14 +30,19 @@ jobs:
           php-version: ${{ matrix.php-version }}
           tools: composer:v2
 
+      - name: Get composer cache directory
+        id: composer-cache
+        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
```

Comment:
`~/.cache/composer` is Linux/macOS specific. This different approach accounts for Windows too.